### ISSUE TITLE
'compileDebugJavaWithJavac' task and 'compileDebugKotlin' task jvm target compatibility should be set to the same Java version

### ISSUE THREAD

**Issue by MatsG23** [spencerccf/app_settings#169]
When I try to run [this Flutter app](https://github.com/nucleus-ffm/foss_warn/tree/broken), I get the following error:
```
Execution failed for task ':app_settings:compileDebugKotlin'.
> 'compileDebugJavaWithJavac' task (current target is 1.8) and 'compileDebugKotlin' task (current target is 11) jvm target compatibility should be set to the same Java version.
  Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain
```

I am using Gradle 8.0.2 and the latest Flutter version in the master release channel (the fix for using Gradle 8 is not in stable yet).

**Comment by navaronbracke** [comment]
@MatsG23 This should have been fixed in `4.3.0`, where I updated `app_settings` to Gradle 8.

**Comment by navaronbracke** [comment]
@MatsG23 I'm not sure when that version will be released either. You can always use a `git` override in your pubspec as a temporary fix:

```
dependencies:
  # TODO: switch to regular version once 4.3.0 is published
  app_settings:
    git: https://github.com/spencerccf/app_settings.git
```
